docs/src/opf/dcp.md

```diff
@@ -1,75 +1,80 @@
 # DC-OPF
 
-See [`DCPPowerModel`](https://lanl-ansi.github.io/PowerModels.jl/stable/formulation-details/#PowerModels.DCPPowerModel) formulation in [`PowerModels.jl`](https://lanl-ansi.github.io/PowerModels.jl/stable/).
+## Mathematical Formulation
 
-## Mathematical formulation
-
-The primal problem reads
+The DCOPF model considered in OPFGenerator is presented below.
 
 ```math
 \begin{align}
-    \min_{\mathbf{p}^{\text{g}}, \mathbf{pf}, \boldsymbol{\theta}} \quad 
-    & \label{eq:DCP:objective}
-        \sum_{g \in \mathcal{G}} c_{g} \mathbf{p}^{\text{g}}_{g} + c^{0}_{g}
-        \\
+    \min_{\PG, \PF, \VA} &\quad
+        \sum_{i \in \NODES} \sum_{j \in \GENERATORS_{i}} c_j \PG_j + c_0 \label{model:dcopf:obj} \\
     \text{s.t.} \quad
-    & \label{eq:DCP:slack_bus}
-        \boldsymbol{\theta}_{\text{slack}} = 0
-        &
-        &&& [\lambda^{\text{slack}}]\\
-    & \label{eq:DCP:kirchhoff}
-        \sum_{g \in \mathcal{G}_{i}} \mathbf{p}^{\text{g}}_{g} 
-        - \sum_{e \in \mathcal{E}^{+}_{i}} \mathbf{pf}_{e}
-        + \sum_{e \in \mathcal{E}^{-}_{i}} \mathbf{pf}_{e}
-        = 
-        \sum_{l \in \mathcal{L}_{i}} p^{d}_{l}
-        + \sum_{s \in \mathcal{S}_{i}} g^{s}_{s}
-        & \forall i \in \mathcal{N}
-        &&& [\lambda^{\text{kcl}}]\\
-    & \label{eq:DCP:ohm}
-        \mathbf{pf}_{e}
-        =
-        b_{e} (\boldsymbol{\theta}_{t(e)} - \boldsymbol{\theta}_{s(e)})
-        & \forall e \in \mathcal{E}
-        &&& [\lambda^{\text{ohm}}]\\
-    & \label{eq:DCP:va_diff}
-        \underline{\Delta \boldsymbol{\theta}}_{e}
-        \leq
-        \boldsymbol{\theta}_{t(e)} - \boldsymbol{\theta}_{s(e)}
-        \leq 
-        \overline{\Delta \boldsymbol{\theta}}_{e}
-        & \forall e \in \mathcal{E}
-        &&& [\mu^{\Delta \boldsymbol{\theta}}]\\
-    & \label{eq:DCP:thermal}
-        -\overline{f}_{e} \leq \mathbf{pf}_{e} \leq \overline{f}_{e}
-        & \forall e \in \mathcal{E}
-        &&& [\mu^{f}]\\
-    & \label{eq:DCP:pg_bounds}
-        \underline{p}^{g}_{g} \leq \mathbf{p}^{\text{g}}_{g} \leq \overline{p}^{g}_{g}
-        & \forall g \in \mathcal{G}
-        &&& [\mu^{pg}]
+    & \sum_{j\in\GENERATORS_i}\PG_j - \sum_{e \in \mathcal{E}_{i}}  \PF_{e} + \sum_{e \in \mathcal{E}^R_{i}} \PF_{e}
```

Comment:
I suggest the use of leaving/entering arcs rather than "reverse" branch. I think the latter is more power systems-centric and less intuitive/common for non experts.

```suggestion
    & \sum_{j\in\GENERATORS_i}\PG_j - \sum_{e \in \mathcal{E}^{+}_{i}}  \PF_{e} + \sum_{e \in \mathcal{E}^{-}_{i}} \PF_{e}
```